docs/draft/mentions.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Mentions
+nav_order: 2
+has_children: true
+permalink: /specs/draft
+---
+
+# Mentions
+
+## Usage
+
+To mention a user, one would type a message containing `@0xpk` where `pk` is the public key of the user to be mentioned.
+
+## Implementation in status-go
+
+`status-go` parses the markdown in the messages and detects mentions that follow the pattern described above. This is done in [`parser.inline.go`](https://github.com/status-im/markdown/blob/b9fe921681227b1dace4b56364e15edb3b698308/parser/inline.go#L68). When mentions are found they are added as an element to the message that is passed to the client with the following format:
+
+```
+{:type "mention" :literal "0x......"}
+```
+
+The `status-react` client uses specific styling for the following elements: `code`, `emph`, `strong`, `link`, `status-tag` and `mention`.
+`mentions` will be displayed in a color that sets it apart from regular text and tapping it will open the chat with the mentioned user.
+
+## Future work
+
+The current implementation is cumbersome as it forces users to input the whole public key of the mentioned user. Implementing auto-completion of mentions would make the feature more appealing to users. It would allow them to mention other users in the chat by usernames or random usernames, which would be replaced by the pk before sending the message
```

Comment:
I initially thought it autocompleted and worked with ENS names too

Does that mean this is very much a WIP feature and should remain a draft for the time being?

cc @andremedeiros @hesterbruikman

Reply:
yes actually mentions is my next assignment so I'm going to update this draft afterward